lexicons/com/atproto/admin/defs.json

```diff
@@ -93,16 +89,35 @@
     },
     "actionReversal": {
       "type": "object",
-      "required": ["reason", "createdBy", "createdAt"],
+      "required": ["createdBy", "createdAt"],
       "properties": {
-        "reason": { "type": "string" },
+        "comment": { "type": "string" },
         "createdBy": { "type": "string", "format": "did" },
         "createdAt": { "type": "string", "format": "datetime" }
       }
     },
+    "actionMeta": {
+      "type": "object",
+      "properties": {
+        "resolveReportIds": { "type": "array", "items": { "type": "integer" } },
+        "reportType": {
+          "type": "ref",
+          "ref": "com.atproto.moderation.defs#reasonType"
+        }
+      }
+    },
     "actionType": {
       "type": "string",
-      "knownValues": ["#takedown", "#flag", "#acknowledge", "#escalate"]
+      "knownValues": [
+        "#takedown",
+        "#flag",
+        "#acknowledge",
+        "#escalate",
+        "#comment",
+        "#label",
+        "#revert",
+        "#mute"
+      ]
     },
```

Comment:
Does the action type imply what content is allowed in the action?  E.g. can you include labels in a `#mute` event even though there's also a `#label` event?  If so, we might consider turning actions into an open union.

Reply:
I want to keep this flexible for now and not apply too much validation based on action types just yet. Some of these props are kinda composable. for instance, I could #acknowledge for a certain period of time and the action will be reverted after that period or #mute AND apply a label. some are not so composable for instance a #revert with a expiry probably doesn't make too much sense.
so overall, there should be some specific combinations that are not allowed but I haven't thought too much into it yet.